### PR TITLE
Adding 'table of Contents' to readme

### PR DESCRIPTION
**Pull request by the94air**
Please merge it this time. It looks so clean and organized.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,32 @@
-# python-digitalocean
-
-This library provides easy access to Digital Ocean APIs to deploy droplets, images and more.
-
-[![](https://travis-ci.org/koalalorenzo/python-digitalocean.svg)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-
-[![](https://img.shields.io/github/forks/badges/shields.svg?style=social&label=Fork)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-[![](https://img.shields.io/github/stars/badges/shields.svg?style=social&label=Star)](https://travis-ci.org/koalalorenzo/python-digitalocean)
-[![](https://img.shields.io/github/watchers/badges/shields.svg?style=social&label=Watch)](https://travis-ci.org/koalalorenzo/python-digitalocean)
+<h1 align="center">python-digitalocean</h1>
```
Comment (koalalorenzo) [inline README.md]: Please use Markdown.

**Comment by koalalorenzo** [comment]
This looks like a better PR than the previous one! Happy to see that 👍 well done!

**Comment by koalalorenzo** [comment]
Can you also pull the master changes into your branch, to resolve conflicts?

File: README.md (hunk 1 above)
Comment (the94air) [reply README.md]: Unfortunately markdown does not support centering. I have to use HTML for this.

**Comment by koalalorenzo** [comment]
Could you resolve the conflicts? 👍 (Maybe rebase/merge?)

**Comment by the94air** [comment]
I don't know what the problem. `git pull` but my branch is up to date with master.

**Comment by koalalorenzo** [comment]
Github is reporting conflicts 🤷‍♂️

**Comment by the94air** [comment]
Wow. Markdown is acting weird today.

**Comment by koalalorenzo** [comment]
Please, let me know when it is ready :)

**Comment by the94air** [comment]
Done!

**Comment by the94air** [comment]
Thanks!.

**Pull request merged**